.gitignore

```diff
@@ -1,3 +1,4 @@
 /target/
 /jaxb-api/target/
 /jaxb-api-test/target/
+/spec/target/
```

Comment:
Trailing space was left here. :)
Also all those target directories can be shrinked under a single pattern: **/target/